Src/BootCamp.Chapter/Area51.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BootCamp.Chapter
+{
+    public class Area51
+    {
+        private List<Credentials> loggedInAccounts = new List<Credentials>();
+        readonly CredentialsManager credentialsManager = new CredentialsManager("Login.txt");
+        const string login = "login";
+        const string create = "create";
+        const string see = "see";
+        const string quit = "quit";
+        public void Demo()
+        {
+            string answer = string.Empty;
+            while (answer != quit)
+            {
+                answer = GetAnswerMainMenu();
+                GoToPartOfProgram(answer);
+            }
+        }
+        private string GetAnswerMainMenu()
+        {
+
+            string input = string.Empty;
+            bool isLoopMainMenu = true;
+
+            while (isLoopMainMenu)
+            {
+                Console.Clear();
+                Console.WriteLine("Login\r\nCreate\r\nSee\r\nQuit");
+                Console.Write("Please choose one: ");
+                input = Console.ReadLine();
+                isLoopMainMenu = AnswerParser(input);
+            }
+            return input.ToLower();
+
+        }
+        private void GoToPartOfProgram(string input)
+        {
+            switch (input)
+            {
+                case login:
+                    Login();
+                    break;
+                case create:
+                    CreateAccount();
+                    break;
+                case see:
+                    See();
+                    break;
+                default:
+                    Console.Clear();
+                    Console.WriteLine("Terminating Program!");
+                    Console.ReadKey();
+                    break;
+            }
+        }
+        private void Login()
+        {
+            bool isKeepLoopingLogin = true;
+
+            while (isKeepLoopingLogin)
+            {
+                Console.Clear();
+                Console.WriteLine("Create Account.");
+                Console.Write("Login Name:");
+                string username = Console.ReadLine();
+
+                Console.Write("Password:");
+                string password = Console.ReadLine();
+                try
+                {
+                    if (Credentials.TryParse(username + "," + password, out Credentials credentials)
+                    && credentialsManager.Login(credentials))
+                    {
+                        Console.Clear();
+                        Console.WriteLine($"Logging {username} in now.");
```

Comment:
This could probably be a part of credentialManagers login method (only these 3 lines)